test/helpers.js

```diff
@@ -2,8 +2,7 @@
 /* eslint no-global-assign: "off" */
 import { DateTime, Settings } from "../src/luxon";
 
-
-exports.withoutIntl = function (name, f) {
+exports.withoutIntl = function(name, f) {
```

Comment:
These changes were unintentional and must have been the linter? I can remove if necessary